test/support/fixtures/tachyon.ex

```diff
@@ -0,0 +1,2 @@
+defmodule Teiserver.TachyonFixtures do
```

Comment:
This is for future use?

Reply:
ooops, that's something I have in the pipes and should not have been commited, removed.